HealthMonitoring.Integration.PushClient.UnitTests/EndpointNotifierTests.cs

```diff
@@ -34,7 +34,7 @@ public class EndpointNotifierTests
             var minimumChecks = 3;
             var expectedEventTimeline = new List<string>();
             for (var i = 0; i < minimumChecks; ++i)
-                expectedEventTimeline.AddRange(new[] { "updateHealth_start", "delay_start", "update_finish", "update_finish" });
+                expectedEventTimeline.AddRange(new[] { "update_start", "update_finish" });
```

Comment:
In the test version on left side, 2 things were checked:

1. that within the attempt, the update health and delay starts at the same time and finish independently (however both have to finish in order to finish the iteration/round). So the valid pattern is SSFF (while SFSF is invalid).
  This check covers the scenario where if update takes 20s and sleep 30s, the round would finish in ~30s (so take the longer action time). Technically it checks also that update starts first, then delay (but probably this is not that important).

2. that the second round will start when first one is finished (so both tasks are awaited, even if one is already finished)
  This ensures that the updates are performed withing the specified, more or less constant intervals.

The test on the right, is much more relaxed as it will more or less check that next round would start when previous is finish (so pattern like S??F S??F) but nothing more. The test coverage from point 1 is lost with this change.